prime-router/settings/wa-doh.yml

```
@@ -0,0 +1,23 @@
+---
+-  name: wa-doh
+  description: Washington State Department of Health
+  jurisdiction: STATE
+  stateCode: WA
+  receivers:
+      - name: elr-download
+        organizationName: ca-scc-phd
+        topic: covid-19
+        jurisdictionalFilter: [ "matches(ordering_facility_state, WA)" ]
+        deidentify: false
+        translation:
+          type: HL7
+          useBatchHeaders: false
+          receivingApplicationName: WADOHPHRED
+          receivingApplicationOID: 2.16.840.1.113883.3.237.4.2
+          receivingFacilityName: dn1fro00
+          receivingFacilityOID: 1.3.6.1.4.1.38630.2.1.1.19
+        timing:
+          operation: MERGE
+          numberPerDay: 1 # Every day
+          initialTime: 00:00
```

```suggestion
          initialTime: 09:15
```

I'm trying to do all our one-per-day deliveries at 09:15am ET

---

Ok.